### PR TITLE
add player prompt

### PR DESCRIPTION
**Pull request by kudukacar**
Indicate's the player's turn in the prompt

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -31,12 +29,14 @@ def in_progress?
...
-  private
-
   def next_token
     grid.count(tokens[:X]) > grid.count(tokens[:O]) ? tokens[:O] : tokens[:X]
   end
 
+  private
+
+  attr_reader :grid, :tokens
+
```
Comment (damonkelley) [inline lib/board.rb:39]: ❤️

File: lib/player.rb [hunk 2]
```diff
@@ -12,6 +13,7 @@ def initialize(display)
   end
 
   def selection(board)
+    display.output(MESSAGES[:player_prompt] + board.next_token)
```
Comment (damonkelley) [inline lib/player.rb]: Is there a way we could make this more intention revealing?

File: spec/player_spec.rb [hunk 3]
```diff
@@ -21,27 +21,31 @@ def input
...
 
-    def initialize(grid)
-      @grid = grid
+    def initialize
+      @selection = 1
     end
 
     def is_available?(position)
-      ![1].include?(position)
+      (1..9).cover?(position) unless position == selection
```
Comment (damonkelley) [inline spec/player_spec.rb:31]: TIL about `#cover?`! 🙌

**Review by damonkelley — COMMENTED** [review]
Looks good!

I am a little bummed that it requires `#next_token` to become public, but that is ok. How many public methods does it have now?

File: spec/player_spec.rb (hunk 3 above)
Comment (kudukacar) [reply spec/player_spec.rb:31]: rubocop not me

File: lib/player.rb (hunk 2 above)
Comment (kudukacar) [reply lib/player.rb]: Sure

File: lib/board.rb (hunk 1 above)
Comment (kudukacar) [reply lib/board.rb:39]: Too many but that could be the nature of the board as the focus of the game

**Pull request merged**